gwt/src/main/resources/BUILD

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2017 The Dagger Authors.
```

Comment:
I think this should just be `//gwt/BUILD`. Does that mess with the location of the `.gwt.xml` files?